ref/config/serverConfiguration.adoc

```diff
@@ -24,171 +18,134 @@ following example:
+. `${server.config.dir}/configDropins/overrides/`
 
-The `${server.config.dir}/server.xml` file must be present but the other files
-are optional. This allows a very flexible way to compose config by simply
-dropping server-formatted XML files into directories. Within the two
-`configDropins` directories the files are read in alphabetical order.
+The `${server.config.dir}/server.xml` file must be present, but the other files are optional. You can flexibly compose config by dropping server-formatted XML files into directories. Files are read in alphabetical order in each of the two `configDropins` directories.
 
-=== Variable substitution
-You can parameterize server config using variables. When resolving
-variable names the following sources are consulted in increasing order of
-precedence:
+== Variable substitution
+You can use variables to parameterize the server configuration. To resolve variable names, the following sources are consulted:
 
-* `server.xml` default variable values
-* environment variables
+* `server.xml` file default variable values
+* Environment variables
 * `bootstrap.properties`
 * Java system properties
-* `server.xml` config
+* `server.xml` configuration
 
-Variables are referenced using `${variableName}` syntax. In server config,
-specify variables using the `variable` element:
+In this list, each source takes precedence over the previous source, so that the `server.xml` configuration takes the highest precedence and the  `server.xml` file default variable values take the lowest precedence. Variables are referenced by using the `${variableName}` syntax. You can specify variables in the server configuration by using the  `variable` element, as shown in the following example:
 
 [source,xml]
 ----
 <variable name="variableName" value="some.value" />
 ----
 
-Default values, specified in server config, are only used if no other value can
-be found. They are specified using the `variable` element and the `defaultValue`
-attribute:
+Default values, which are specified in the `server.xml` file, are used only if no other value is specified. These values are specified by using the `variable` element and the `defaultValue` attribute:
 
 [source,xml]
 ----
 <variable name="variableName" defaultValue="some.default.value"/>
 ----
 
-Environment variables can be accessed as variables. From 19.0.0.3, they can be
-accessed directly by referencing the environment variable name. If the variable
-cannot be resolved the following transformations on the environment variable
-name is tried:
+Environment variables can be accessed as variables. As of version 19.0.0.3, you can reference the environment variable name directly. If the variable cannot be resolved as specified, the `server.xml` file looks for the following variations on the environment variable name:
 
-1. Replace all non-alpha num characters with _
-2. Change all characters to upper case.
+* Replacing all non-alphanumeric characters with the underscore character (`_`)
```

Comment:
I don't think using ing words makes it clearer what is going on.

Reply:
Updated